packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -297,6 +297,10 @@ class TextMapPropagator {
   }
 
   _extractTraceparentContext (carrier) {
+    if (carrier.hasOwnProperty('x-cloud-trace-context')) {
```

Comment:
Google sets `x-cloud-trace-context` and `traceparent` headers when our cloud function is triggered, used for Google Cloud Tracing.

This causes the tracer to set the `parent_id` of our cloud function trace's root span, thereby causing the span to not show up under the `Trace Root Spans` in Datadog.

This prevents the traceparent context from being used to set the span context, so cloud function trace root spans show up properly as trace root spans.